contrib/hawq-ambari-plugin/build.properties

```diff
@@ -0,0 +1,8 @@
+hawq.release.version=2.0.1
+hawq.common.services.version=2.0.0
+pxf.release.version=3.0.1
+pxf.common.services.version=3.0.0
+hawq.repo.prefix=hawq
+hawq.addons.repo.prefix=hawq-add-ons
+repository.version=2.0.1.0
+default.stack=HDP-2.4
```

Comment:
There is no major problem with putting these parameters back in the pom.xml

I find it to be cleaner and better managed as a separate file. 

For the commercial release, a different set of build.properties can be used. It is easier to use a new build.properties file rather than updating the pom.xml or explicitly specify overrides for each parameter in the mvn command during the build process. 

Do you still want it back in the pom.xml?